file/lib/file_source_endpoint.ex

```diff
@@ -29,21 +32,40 @@ defmodule Membrane.RTC.Engine.Support.FileSourceEndpoint do
               ],
               ssrc: [
                 spec: RTP.ssrc_t(),
+                default: 1,
                 description: "SSRC of RTP packets"
               ],
               payload_type: [
                 spec: RTP.payload_type_t(),
+                defalt: 127,
                 description: "Payload type of RTP packets"
+              ],
+              toilet_capacity: [
+                spec: integer(),
+                default: 1_000,
+                description:
+                  "Size of toilet between payloader and realtimer and between realtimer and track_sender"
+              ],
+              after_source_transformation: [
+                spec: (ChildrenSpec.builder(), Track.t() -> ChildrenSpec.builder()),
+                default: &__MODULE__.identitiy_transformation/1,
+                description:
+                  "Additional pipeline transformation after `file_source` the output stream must be encoded in OPUS or H264"
```

Comment:
I would expand this description a bit to provide more context, maybe include an example of usage